### PR TITLE
Add Giveth

### PR DESCRIPTION
**Pull request by fleupold**
https://cowservices.slack.com/archives/C04DV3TL1MH/p1671036044310869

### CODE REVIEW

File: src/public/CowSwap.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
   "name": "CoW Swap",
-  "timestamp": "2022-03-01T14:54:18+00:00",
+  "timestamp": "2022-12-015T19:00:00+00:00",
   "version": {
-    "major": 0,
+    "major": 1,
```
Comment (anxolin) [inline src/public/CowSwap.json:5]: Should we increment major if it's just adding a token? 

See https://github.com/Uniswap/token-lists#semantic-versioning

**Review by anxolin — COMMENTED** [review]
Looks good, I'm just not sure about the semversioning. 

Out of curiosity, did you manage to enable versioning in S3?

File: src/public/CowSwap.json (hunk 1 above)
Comment (fleupold) [reply src/public/CowSwap.json:5]: This was rebased on the wrong version of main. The current version is [1.0.0](https://files.cow.fi/tokens/CowSwap.json) and so adding a token should just increment the patch to 1.0.1. Will rebase on top of main

**Comment by fleupold** [comment]
> Out of curiosity, did you manage to enable versioning in S3?

yes

**Pull request merged**